canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -320,6 +331,9 @@ class ToolTip extends React.Component {
 		// To prevent this default behavior, stopPropagation and preventDefault is used.
 		evt.stopPropagation();
 		evt.preventDefault();
+
+		// when tooltip with link is closed and another tooltip is opened newly opened tooltip should have focus
```

Comment:
Please start sentences in the comments with a capital letter. If you start with a lowercase letter it looks like part of the sentence has been lost.